packages/access-policy/src/AccessPolicy.js

```diff
@@ -15,9 +15,6 @@ export class AccessPolicy {
       throw new Error('Cannot instantiate an AccessPolicy without providing the policy details');
     }
     const permissionGroupLists = Object.values(this.policy);
-    if (permissionGroupLists.length === 0) {
-      throw new Error('At least one entity should be specified in an access policy');
-    }
```

Comment:
Removing this restriction since we're now supporting users having no access at first registration.